src/main/scala/scorex/waves/http/DebugApiRoute.scala

```diff
@@ -74,6 +76,40 @@ case class DebugApiRoute(settings: RestAPISettings,
     )
   }
 
+  @Path("/portfolios/{address}")
+  @ApiOperation(
+    value = "Portfolio",
+    notes = "Get current portfolio considering pessimistic transactions in the UTX pool",
+    httpMethod = "GET"
+  )
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(
+      name = "address",
+      value = "An address of portfolio",
+      required = true,
+      dataType = "string",
+      paramType = "path"
+    ),
+    new ApiImplicitParam(
+      name = "considerUnspent",
+      value = "Taking into account pessimistic transactions from UTX pool",
+      required = false,
+      dataType = "boolean",
+      paramType = "query",
+      defaultValue = "true"
+    )
+  ))
+  @ApiResponses(Array(new ApiResponse(code = 200, message = "Json portfolio")))
+  def portfolios: Route = path("portfolios" / Segment) { (rawAddress) =>
+    (get & parameter('considerUnspent.as[Boolean])) { (considerUnspent) =>
```

Comment:
I think, QAs should make this calculation by themselves, it is not too hard. If we really need this, i'd rather create a special API case class and make a [de]serializers for it.